docs/tutorials/prometheus.ipynb

```diff
@@ -0,0 +1,471 @@
+        "In order to show some activity, `dig` command could be used to generate a few DNS queries against the CoreDNS server that has been setup:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "7XLuWyE3Pgg0",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!apt-get install -y -qq dnsutils\n",
+        "\n",
+        "!dig @127.0.0.1 -p 9053 demo1.example.org\n",
+        "!dig @127.0.0.1 -p 9053 demo2.example.org"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Afb01zkACsuW",
+        "colab_type": "text"
+      },
+      "source": [
+        "The next step is to setup Prometheus server and use Prometheus to scrape CoreDNS metrics that are exposed on port `9153` from above. The `prometheus.yml` file for configuration is also available for [download](https://github.com/tensorflow/io/blob/master/docs/tutorials/prometheus/prometheus.yml):\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "3HHfVdv9D3Jv",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "source": [
+        "!curl -s -OL https://github.com/prometheus/prometheus/releases/download/v2.15.2/prometheus-2.15.2.linux-amd64.tar.gz\n",
+        "!tar -xzf prometheus-2.15.2.linux-amd64.tar.gz --strip-components=1\n",
+        "\n",
+        "!curl -s -OL https://raw.githubusercontent.com/tensorflow/io/master/docs/tutorials/prometheus/prometheus.yml\n",
+        "\n",
+        "# Run `./prometheus` as a background process.\n",
+        "# IPython doesn't recognize `&` in inline bash cells.\n",
+        "get_ipython().system_raw('./prometheus &')"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "f61fK3bXQH4N",
+        "colab_type": "text"
+      },
+      "source": [
+        "Now a CoreDNS server whose metrics are scraped by a Prometheus server and ready to be consumed by TensorFlow."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "acEST3amdyDI",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Create Dataset for CoreDNS metrics and use it in TensorFlow\n",
+        "\n",
+        "Create a Dataset for CoreDNS metrics that is available from PostgreSQL server, could be done with `tfio.experimental.IODataset.from_prometheus`. At the minimium two arguments are needed. `query` is passed to Prometheus server to select the metrics and `length` is the period we want to load into Dataset.\n",
+        "\n",
+        "You can start with \"coredns_dns_request_count_total\" and \"5\" (secs) to create the Dataset below. Since earlier in the tutorial two DNS queries were sent, it is expected that the metrics for \"coredns_dns_request_count_total\" will be \"2.0\" at the end of the time series:"
```

Comment:
```suggestion
        "You can start with `\"coredns_dns_request_count_total\"` and `\"5\"` (secs) to create the Dataset below. Since earlier in the tutorial two DNS queries were sent, it is expected that the metrics for `\"coredns_dns_request_count_total\"` will be `\"2.0\"` at the end of the time series:"
```